nuxeo-coldstorage-web/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "scripts": {
     "postinstall": "check-engine || true",
-    "eslint": "eslint *.html demo/**/*.html test/**/*.html",
+    "eslint": "eslint . --ext html",
     "polylint": "polymer lint",
```

Comment:
comparing to https://github.com/nuxeo/nuxeo-elements/blob/maintenance-2.4.x/package.json#L19 should we keep the `*.html`